app/webpacker/styles/primary-nav.scss

```diff
@@ -0,0 +1,19 @@
+.primary-nav-container {
+  display: flex;
+  flex-direction: row;
+  background-color: govuk-colour("mid-grey");
+  height: 4vh;
```

Comment:
Are we sure we want '4% of available viewport height' here? Can we use rems or pixels?